imap_processing/cdf/cdf_attribute_manager.py

```diff
@@ -93,7 +95,9 @@ def _load_default_global_attr_schema(self) -> dict:
             self.source_dir / "shared" / DEFAULT_GLOBAL_CDF_ATTRS_SCHEMA_FILE
         )
         # Load the Schema
-        return CdfAttributeManager._load_yaml_data(default_schema_path)
+        return CdfAttributeManager._load_yaml_data(default_schema_path)  # type: ignore[no-any-return]
```

Comment:
I am not quite sure what to do about this. My understanding is that these yaml files get converted to dictionaries, and the keys of those dictionaries could be anything, in which case saying the type is "Any" is perfectly valid.

Because of this, I am inclined to add `no-any-return` to the list of disabled error codes.

Curious to know what @maxinelasp thinks about this.

Reply:
The keys of the dictionaries are expected to be `string` types always, with the values either being `string` or `dict`. If we can add that type then it should fix the issue. If not, then `any` type is ok here.

Reply:
I can `from typing import Any` which can allow me to do `self.variable_attributes: dict[Any]`. Doing this way allows for the `no-any-return` error to still be allowed. 
However, `self.variable_attributes: dict[str, dict]` does work, so long as this is the right format for the dict.

Reply:
@maxinelasp and @bourque 
I can import `Any` from typing and then declare the function to return Any which will not trigger any errors.  I can also tell the function to return `yaml` since that is technically what the `_load_yaml_data()` returns that is being called.